content/collections/audit-works.html

```diff
@@ -1,3 +1,262 @@
+<item>
+  <field name="logo">
+    sommelier.png
+  </field>
+  <field name="title">
+    Sommelier-12
+  </field>
+  <field name="subtitle">
+    Automated DeFi yield optimization strategies
+  </field>
+  <field name="date">
+    2023-10-27
+  </field>
+  <field name="about">
+  </field>
+  <field name="report">
+    /library/audits/sommelier-12
+  </field>
+  <field name="attachments">
+    [Repository](https://github.com/PeggyJV/cellar-contracts)
+  </field>
+</item>
+
+<item>
+  <field name="logo">
+    sommelier.png
+  </field>
+  <field name="title">
+    Sommelier-11
+  </field>
+  <field name="subtitle">
+    Automated DeFi yield optimization strategies
+  </field>
+  <field name="date">
+    2023-08-30
+  </field>
+  <field name="about">
```

Comment:
Audited the addition of Frax collateral and debt adaptors, and changes to the AxelarProxy contract.